modules/features/kids/src/main/java/au/com/shiftyjelly/pocketcasts/kids/KidsBottomSheetDialog.kt

```diff
@@ -0,0 +1,72 @@
+package au.com.shiftyjelly.pocketcasts.kids
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.ComposeView
+import androidx.core.view.doOnLayout
+import androidx.fragment.app.viewModels
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.kids.viewmodel.KidsSendFeedbackViewModel
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class KidsBottomSheetDialog : BottomSheetDialogFragment() {
+
+    @Inject
+    lateinit var theme: Theme
+
+    private val viewModel: KidsSendFeedbackViewModel by viewModels()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        val context = context ?: throw Exception("Context not found")
```

Comment:
There is a `requireContext()` function available for fragments. That does exactly that.